tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_delete_app.py

```diff
@@ -46,7 +50,11 @@ def test_delete(self, app_to_delete, install_method):
         # We can't pass parameters to the setUp(). Bug # to be filed
         self.parameterized_set_up(app_to_delete, install_method)
 
-        self.homescreen.delete_app(self.app_to_delete['url']).tap_confirm()
+        confirm_dialog = self.homescreen.delete_app(self.app_to_delete['url']).tap_remove()
+        confirm_dialog_ok = confirm_dialog.tap_confirm()
+       
+        #confirm_dialog_ok.tap_done()
```

Comment:
I try to tap here in order to finish with the first test completely but cannot find that button, although the locator seems easy and clear